spec/requests/api/v0/podcasts_episodes_spec.rb

```diff
@@ -88,16 +88,16 @@
         expect(response.parsed_body.map { |pe| pe["id"] }).to eq([pe1.id])
       end
 
-      it "returns not found if the podcast is unavailable" do
+      it "returns not found if the episode belongs to an unpublished podcast" do
         unavailable_podcast = create(:podcast, published: false)
-        create(:podcast_episode, podcast: unavailable_podcast)
+        pe = create(:podcast_episode, podcast: unavailable_podcast)
 
-        get api_podcast_episodes_path(username: unavailable_podcast.slug)
+        get api_podcast_episodes_path(username: pe.slug)
```

Comment:
It used to be a podcast slug, not an episode slug, I suppose it should remain as it was because we are looking for a podcast by slug in the controller.

Reply:
great catch! thanks! i'll fix it